routes/web.php

```diff
@@ -14,5 +14,18 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('home');
+});
+
+Route::get('/template', function () {
+    return view('layout/app');
+});
+
+Route::get('/theWideMouthFrog', function () {
```

Comment:
I would suggest not using camel case in uri / path names. Those are case-sensitive. Common practice is using the [kebab case](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Why-you-should-make-kebab-case-a-URL-naming-convention-best-practice)

Reply:
Changes have been made